plugin/src/main/java/io/fabric8/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -234,13 +228,8 @@
     public ResourceMojo() {
     }
 
-    @Override
-    public void execute() throws MojoExecutionException, MojoFailureException {
-        clusterAccess = new ClusterAccess(namespace);
```

Comment:
There was a reason to put it that early and not into `executeInternal()`. Just cant remember it but will check it ...